src/opts/config.rs

```diff
@@ -66,9 +66,10 @@ impl Default for LinesToScroll {
     }
 }
 
-#[derive(Deserialize, Debug, Default, PartialEq)]
+#[derive(Deserialize, Clone, Debug, Default, PartialEq)]
 pub struct KeybindingsSection {
-    pub base: Option<Keybindings>,
+    #[serde(default)]
+    pub base: Keybindings,
```

Comment:
This should be the exact same as before but with added benefits:

- `base` is always something, no need to `unwrap` etc...
- if the section doesn't exist, defaults are loaded. This avoids using `unwrap_or_default` everywhere which is error prone imo and simpler to read.
- The app always has `base` keybindings so it makes sense semantically

doc: https://serde.rs/field-attrs.html#default